cypress/integration/rancid_app_spec.js

```diff
@@ -1,101 +1,217 @@
+            body: movies
+          })
+        });
+
+      cy.visit(baseURL);
+    });
+
     it('Should have a title/logo', () => {
-      cy.get('h1').contains('Radioactive Toothbrushes')
+      cy.get('h1').contains('Radioactive Toothbrushes');
     });
 
     it('Should click home icon to navigate back to main', () => {
-      cy.get('section > article')
+      cy.get('section > a')
         .contains('Mulan')
-        .click()
-      cy.get('header > img').click();
+        .click();
+
+      cy.get('header > a').click();
+    });
+  });
+
+  describe('RT Movies', () => {
+    beforeEach(() => {
+      cy.fixture('Film-data.json')
+        .then(movie => {
+          cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919', {
+            body: movie
+          })
+        });
+
+      cy.fixture('Movies-data.json')
+        .then(movies => {
+          cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/', {
+            body: movies
+          })
+        });
+
+      cy.visit(baseURL);
+    });
+
+    it('Should display many movie cards', () => {
+      cy.get('.movies-container').should(($article) => {
+        expect($article).to.have.length;
+
+        expect($article.first()).to.contain('Money Plane');
+      });
+    });
+
+    it('Should contain clickable movie cards', () => {
+      cy.get('.movies-container > a').contains('Money Plane').click();
+    });
+  });
+
+  describe('RT Card', () => {
+    beforeEach(() => {
+      cy.fixture('Movies-data.json')
+        .then(movies => {
+          cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/', {
+            body: movies
+          })
+        });
+
+      cy.visit(baseURL);
+    });
+
+    it('Should display a movie card with an id', () => {
+        cy.get('.card').should('have.attr', 'id');
+    });
+
+    it('Should have an img with alt text', () => {
+        cy.get('.card > img')
+          .get('.card > img').should('have.attr', 'alt');
     });
   });
 
   describe('RT Film', () => {
-    it('Should have a loading message', () => {
-      //set up to access Film
-      cy.get('section > article')
-        .contains('Mulan')
-        .click()
-        //inside of Film
-        .get('h2').should('be.visible')
+    beforeEach(() => {
+      cy.fixture('Film-data.json')
+        .then(movie => {
+          cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/337401', {
+            body: movie
+          })
+        });
+
+      cy.fixture('Movies-data.json')
+        .then(movies => {
+          cy.intercept('https://rancid-tomatillos.herokuapp.com/api/v2/movies/', {
+            body: movies
+          })
+        });
+
+      cy.visit(baseURL);
     });
 
-    //how do we test API isn't functioning?
-    // it('Should have an error message', () => {
-    //   cy.get('section > article')
+    // it.only('Should have a loading message', () => {
```

Comment:
This loading test is the final test. 
We've seen that in the browser that the loading message does appear. However, once in Cypress, we don't see it/Cypress skips over it because we're not honing in on the right element or layer. 
We've reached out to our cohort. I've reached out to my rock. I would say the next step is to get Instructor input on what's occuring.